guidelines/index.html

```diff
@@ -1157,6 +1173,10 @@
 				<dd>
 					<p>any sequence where words and paragraphs are presented in an order that does not change the meaning of the content</p>
 				</dd>
+                        <dt><dfn>CSS pixels</dfn></dt>
+                        <dd>
+                              <p>The reference size that browsers use to calculate the how large text and elements appear when presented on screen. The value of a CSS pixel is set as closely as possible to the <a href="https://www.w3.org/TR/css3-values/#reference-pixel">reference pixel</a>, which takes into account the assumed viewing distance. For example, 1px on a TV display at 3.5m away is much larger than 1px on a phone 30cm feet away. A high resolution display would include more physical (device) pixels for each CSS pixel used in the browser's calculations for text and layout size.</p>
```

Comment:
This definition could do with a bit of tightening/tweaking. Possibly something along the lines of

> A CSS pixel is the canonical unit of measure for all lengths and measurements in CSS. This unit is density-independent, and distinct from actual hardware pixels present in a display. User agents and operating systems should ensure that a CSS pixel is set as closely as possible to the <a href="https://www.w3.org/TR/css3-values/#reference-pixel">reference pixel</a>, which takes into account the physical dimensions of the display and the assumed viewing distance (factors which cannot be determined by content authors).